stdlib/public/core/Array.swift

```diff
@@ -1366,21 +1366,35 @@ extension Array {
 }
 
 extension Array {
+  @available(swift, deprecated: 5.1, renamed: "init(unsafeUninitializedCapacity:initializingWith:)")
```

Comment:
This needs to be OS availability, unfortunately. It's not tied to `-swift-version` but to your deployment target.

Reply:
Addressed these in b962048.